attributes/default.rb

```diff
@@ -32,9 +32,15 @@
   default['grafana']['package']['version'] = node['grafana']['version']
   default['grafana']['package']['apt_rebuild'] = true
   default['grafana']['package']['trusted'] = false
-when 'rhel', 'fedora'
-  default['grafana']['package']['repo'] = 'https://packagecloud.io/grafana/stable/el/$releasever/$basearch'
-  default['grafana']['package']['key'] = 'https://grafanarel.s3.amazonaws.com/RPM-GPG-KEY-grafana'
+when 'rhel', 'fedora', 'amazon'
+  if node['platform_family'] == 'amazon'
+    release_version = '7'
```

Comment:
adding condition for amazon since $releasever evaluates to amazonlinux version (ie. '2017.09') which is invalid repo path for 'https://packagecloud.io/grafana/stable/el'

Reply:
Do we want to make amazon to RHEL 7? Amazon 2 is RHEL 7 based, but Amazon 201X is RHEL 6 based?

Reply:
Great point! Yes, want amazon to be rhel 6 based so changed from '7' to '6': 
https://aws.amazon.com/marketplace/pp/B007ORSS8I/ref=pe_redhat_freetier/

Thank you!